styx-service-common/src/test/java/com/spotify/styx/storage/WFIExecutionBuilderTest.java

```diff
@@ -531,7 +527,6 @@ public void testStop() {
         SequenceEvent.create(E.runError("First failure"), c++, ts("07:58")),
         SequenceEvent.create(E.retryAfter(10), c++, ts("07:59")),
 
-        SequenceEvent.create(E.retry(), c++, ts("08:56")),
```

Comment:
There should be a `dequeue`.

Reply:
Covered here https://github.com/spotify/styx/pull/827/files#diff-d145c629c9c6e0c3dd3b1413d819d039R162